--- open-iscsi-2.1.10/libiscsi/Makefile.orig	2025-02-16 18:11:07.852468418 +0100
+++ open-iscsi-2.1.10/libiscsi/Makefile	2025-02-16 18:13:27.081714148 +0100
@@ -19,10 +19,10 @@ CFLAGS += -DSBINDIR=\"$(SBINDIR)\"
 CFLAGS += -DISCSI_DB_ROOT=\"$(DBROOT)\"
 CFLAGS += -DISCSI_VERSION_STR=\"$(ISCSI_VERSION_STR)\"
 
-CFLAGS += -L../redhat-linux-build
+CFLAGS += -L../build
 LDLIBS += -lopeniscsiusr -lkmod -lcrypto
 
-LIB = libiscsi.so.0
+LIB = libopeniscsi.so.0
 TESTS = tests/test_discovery_sendtargets tests/test_discovery_firmware
 TESTS += tests/test_login tests/test_logout tests/test_params
 TESTS += tests/test_get_network_config tests/test_get_initiator_name
