--- open-iscsi-2.1.7/etc/systemd/iscsid.service.template.orig	2022-05-16 22:04:51.668726910 +0200
+++ open-iscsi-2.1.7/etc/systemd/iscsid.service.template	2022-05-16 22:12:39.789524215 +0200
@@ -9,7 +9,7 @@ Requires=iscsi-init.service iscsi-shutdo
 [Service]
 Type=notify
 NotifyAccess=main
-ExecStart=@SBINDIR@/iscsid -f
+ExecStart=@SBINDIR@/iscsid -f -u iscsi -g iscsi
 KillMode=mixed
 Restart=on-failure
 
--- open-iscsi-2.1.7/etc/systemd/iscsi-init.service.template.orig	2022-05-16 22:24:15.152423778 +0200
+++ open-iscsi-2.1.7/etc/systemd/iscsi-init.service.template	2022-05-16 22:39:01.854286768 +0200
@@ -17,4 +17,4 @@ WantedBy=systemd-remount-fs.service
 [Service]
 Type=oneshot
 RemainAfterExit=no
-ExecStart=/usr/bin/sh -c 'echo "InitiatorName=`@SBINDIR@/iscsi-iname`" > /etc/iscsi/initiatorname.iscsi'
+ExecStart=/bin/sh -c 'echo "InitiatorName=`@SBINDIR@/iscsi-iname`" > /etc/iscsi/initiatorname.iscsi'
--- open-iscsi-2.1.10/etc/systemd/iscsi-onboot.service.template.orig	2025-02-16 15:29:11.268441005 +0100
+++ open-iscsi-2.1.10/etc/systemd/iscsi-onboot.service.template	2025-02-16 15:29:15.091753626 +0100
@@ -9,7 +9,7 @@ ConditionDirectoryNotEmpty=/sys/class/is
 
 [Service]
 Type=oneshot
-ExecStart=-/usr/libexec/iscsi-mark-root-nodes
+ExecStart=-/lib/systemd/pld-helpers.d/iscsi-mark-root-nodes
 
 [Install]
 WantedBy=sysinit.target
--- open-iscsi-2.1.10/etc/systemd/iscsi-shutdown.service.template.orig	2025-02-16 14:29:25.464533640 +0100
+++ open-iscsi-2.1.10/etc/systemd/iscsi-shutdown.service.template	2025-02-16 15:29:53.031548089 +0100
@@ -11,5 +11,5 @@ RefuseManualStop=yes
 [Service]
 Type=oneshot
 RemainAfterExit=true
-ExecStart=-/usr/bin/true
+ExecStart=-/bin/true
 ExecStop=-@SBINDIR@/iscsiadm -m node --logoutall=all
--- open-iscsi-2.1.10/etc/systemd/iscsi-starter.service.template.orig	2025-02-16 14:29:25.464533640 +0100
+++ open-iscsi-2.1.10/etc/systemd/iscsi-starter.service.template	2025-02-16 15:30:29.811348835 +0100
@@ -7,7 +7,7 @@ ConditionDirectoryNotEmpty=@DBROOT@/node
 [Service]
 Type=oneshot
 RemainAfterExit=true
-ExecStart=@SBINDIR@/systemctl start --no-block --job-mode=fail iscsi.service
+ExecStart=/bin/systemctl start --no-block --job-mode=fail iscsi.service
 
 [Install]
 WantedBy=sysinit.target
