--- open-iscsi-2.1.10/libiscsi/Makefile.orig	2025-02-16 15:32:07.487486345 +0100
+++ open-iscsi-2.1.10/libiscsi/Makefile	2025-02-16 15:33:32.870357120 +0100
@@ -22,7 +22,7 @@ CFLAGS += -DISCSI_VERSION_STR=\"$(ISCSI_
 CFLAGS += -L../redhat-linux-build
 LDLIBS += -lopeniscsiusr -lkmod -lcrypto
 
-LIB = libiscsi.so.0
+LIB = libopeniscsi.so.0
 TESTS = tests/test_discovery_sendtargets tests/test_discovery_firmware
 TESTS += tests/test_login tests/test_logout tests/test_params
 TESTS += tests/test_get_network_config tests/test_get_initiator_name
