--- open-iscsi-2.1.4/etc/systemd/iscsi.service.template.orig	2022-05-15 21:39:53.531578661 +0200
+++ open-iscsi-2.1.4/etc/systemd/iscsi.service.template	2022-05-15 21:43:54.510273168 +0200
@@ -5,7 +5,7 @@ DefaultDependencies=no
 Before=remote-fs-pre.target
 After=network.target network-online.target iscsid.service iscsiuio.service systemd-remount-fs.service
 Wants=remote-fs-pre.target
-ConditionDirectoryNotEmpty=/var/lib/iscsi/nodes
+ConditionDirectoryNotEmpty=/etc/iscsi/nodes
 
 [Service]
 Type=oneshot
--- open-iscsi-2.1.7/etc/systemd/iscsid.service.template.orig	2022-05-16 22:04:51.668726910 +0200
+++ open-iscsi-2.1.7/etc/systemd/iscsid.service.template	2022-05-16 22:12:39.789524215 +0200
@@ -9,7 +9,7 @@ Requires=iscsi-init.service iscsi-shutdo
 [Service]
 Type=notify
 NotifyAccess=main
-ExecStart=@SBINDIR@/iscsid -f
+ExecStart=@SBINDIR@/iscsid -f -u iscsi -g iscsi
 KillMode=mixed
 Restart=on-failure
 
--- open-iscsi-2.1.7/Makefile.orig	2022-05-16 22:24:15.152423778 +0200
+++ open-iscsi-2.1.7/Makefile	2022-05-16 22:38:37.014421337 +0200
@@ -7,7 +7,7 @@
 DESTDIR ?=
 
 prefix = /usr
-exec_prefix = /usr
+exec_prefix =
 mandir = $(prefix)/share/man
 etcdir = /etc
 
--- open-iscsi-2.1.7/etc/systemd/iscsi-init.service.template.orig	2022-05-16 22:24:15.152423778 +0200
+++ open-iscsi-2.1.7/etc/systemd/iscsi-init.service.template	2022-05-16 22:39:01.854286768 +0200
@@ -17,4 +17,4 @@ WantedBy=systemd-remount-fs.service
 [Service]
 Type=oneshot
 RemainAfterExit=no
-ExecStart=/usr/bin/sh -c 'echo "InitiatorName=`@SBINDIR@/iscsi-iname`" > /etc/iscsi/initiatorname.iscsi'
+ExecStart=/bin/sh -c 'echo "InitiatorName=`@SBINDIR@/iscsi-iname`" > /etc/iscsi/initiatorname.iscsi'
--- open-iscsi-2.1.4/etc/systemd/iscsi-onboot.service.orig	2022-05-15 22:26:25.683118945 +0200
+++ open-iscsi-2.1.4/etc/systemd/iscsi-onboot.service	2022-05-16 20:50:19.019623997 +0200
@@ -9,7 +9,7 @@ ConditionDirectoryNotEmpty=/sys/class/is
 
 [Service]
 Type=oneshot
-ExecStart=-/usr/libexec/iscsi-mark-root-nodes
+ExecStart=-/lib/systemd/pld-helpers.d/iscsi-mark-root-nodes
 
 [Install]
 WantedBy=sysinit.target
--- open-iscsi-2.1.4/etc/systemd/iscsi-shutdown.service.orig	2022-05-15 22:26:25.683118945 +0200
+++ open-iscsi-2.1.4/etc/systemd/iscsi-shutdown.service	2022-05-16 20:50:36.799527675 +0200
@@ -11,5 +11,5 @@ RefuseManualStop=yes
 [Service]
 Type=oneshot
 RemainAfterExit=true
-ExecStart=-/usr/bin/true
-ExecStop=-/usr/sbin/iscsiadm -m node --logoutall=all
+ExecStart=-/bin/true
+ExecStop=-/sbin/iscsiadm -m node --logoutall=all
